source/portable/BufferManagement/BufferAllocation_2.c

```diff
@@ -63,6 +63,10 @@
     #define baMINIMAL_BUFFER_SIZE    sizeof( ARPPacket_t )
 #endif /* ipconfigUSE_TCP == 1 */
 
+#define baALIGNMENT_BYTES            ( sizeof( size_t ) )
+#define baALIGNMENT_MASK             ( sizeof( size_t ) - 1U )
```

Comment:
Can this be made as:
#define baALIGNMENT_MASK  ( baALIGNMENT_BYTES - 1U ).
This will make the baALIGNMENT_MASK and baALIGNMENT_BYTES dependent and we will not have to update both in case there is a change in the first.

Reply:
Sure, will update, thanks